src/hotspot/cpu/s390/templateTable_s390.cpp

```diff
@@ -2416,13 +2418,71 @@ void TemplateTable::load_field_cp_cache_entry(Register obj,
   }
 }
 
+void TemplateTable::load_invokedynamic_entry(Register method) {
+  const Register cache    = Z_tmp_1;
+  const Register index    = Z_tmp_3;
+  const Register appendix = Z_R1_scratch;
+  assert_different_registers(cache, index, appendix, method);
+
+  Label resolved;
+  __ load_resolved_indy_entry(cache, index);
+  __ z_lg(method, Address(cache, in_bytes(ResolvedIndyEntry::method_offset())));
+
+  // The invokedynamic is unresolved iff method is NULL
+  __ z_clgij(method, 0, Assembler::bcondNotEqual, resolved); // method != 0, jump to resolved
```

Comment:
In the light of the ongoing effort to substitute all occurrences of NULL (and (void*)0) with nullptr, you may want to substitute 0 with (unsigned long)nullptr here.